### PR TITLE
feat: styles

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Set up styling

# Changes

- copy global styles from nns-dapp
- add sass
- document import in readme

# Tests

Import library global style and mixin locally in nns-dapp.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,39 @@
...
+
+```bash
+npm i @dfinity/gix-components
+```
+
+## Usage
+
+### Styling
+
+The styles provided by the kit are not pre-compiled - i.e. is not provided as `CSS` files but as `SCSS`. Make sure that a SASS preprocessing is installed on your side.
```
Comment (mstrasinskis) [inline README.md:23]: Does it mean that sass should be installed with `-g`?

File: src/lib/styles/global/fonts.scss [hunk 2]
```diff
@@ -0,0 +1,94 @@
+/* Font sizing source: https://type-scale.com/ */
```
Comment (mstrasinskis) [inline src/lib/styles/global/fonts.scss:1]: Hmm... I didn't find our set on `https://type-scale.com/`. Was it modified?

**Review by mstrasinskis — APPROVED** [review]
LGTM, thanks. A couple of questions.

File: README.md (hunk 1 above)
Comment (peterpeterparker) [reply README.md]: Oh no, just in the consuming project. Modified the sentence from "on your side" to "in your project". Thanks

**Pull request merged**